init-wifi.h

```diff
@@ -103,18 +114,16 @@ bool tryConnectWifi(const char* ssid, const char* pwd) {
      Serial.print(".");
      attempts++;
   }
+  Serial.println();
 
   //!!! ip adress condition part does not work
```

Comment:
if this condition does not work - what about removing it?

Reply:
yep - will do that